uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -386,6 +386,12 @@ <h4 class="modal-title">
         ng-click="$ctrl.cancel()">
         Back
       </button>
+      <a
+        class="text-danger pull-left pointer chr_leave-request-modal__delete"
+        ng-if="$ctrl.isRole('admin') && $ctrl.directiveOptions.leaveRequest"
```

Comment:
shouldn't you check that the request has an `id` rather than testing simply for `$ctrl.directiveOptions.leaveRequest` ?